src/main/kotlin/com/charlatano/scripts/BombTimer.kt

```diff
@@ -44,29 +44,30 @@ fun bombTimer() {
     }
 }
 
-fun currentGameTicks(): Float = CSGO.engineDLL.float(EngineOffsets.dwGlobalVars + 16)
+private fun currentGameTicks(): Float = CSGO.engineDLL.float(EngineOffsets.dwGlobalVars + 16)
```

Comment:
This should be a public method since you can use it on other functions but should move out of the bomb timer.